### PR TITLE
Replace lodash _.find with native Array.find

### PR DESCRIPTION
**Pull request by chrisVillanueva**
Addresses: #7747

### CODE REVIEW

File: lib/plugins/aws/deploy/lib/createStack.js [hunk 1]
```diff
@@ -69,9 +68,10 @@ module.exports = {
               // custom deployment bucket won't generate any output (no check)
               !this.serverless.service.provider.deploymentBucket;
             if (shouldCheckStackOutput) {
-              const isAlreadyAccelerated = !!_.find(data.Stacks[0].Outputs, {
-                OutputKey: 'ServerlessDeploymentBucketAccelerated',
-              });
+              const isAlreadyAccelerated = !!data.Stacks[0].Outputs.find(
```
Comment (medikoo) [inline lib/plugins/aws/deploy/lib/createStack.js:71]: In this case, we can remove `!!` (boolean coercion) and replace `find` with `some`

File: lib/plugins/aws/package/compile/events/sns/index.js [hunk 2]
```diff
@@ -35,7 +35,7 @@ class AwsCompileSNSEvents {
       return false;
     }
     const intrinsicFunctions = ['Fn::ImportValue', 'Ref', 'Fn::GetAtt', 'Fn::Sub', 'Fn::Join'];
-    return !!_.find(intrinsicFunctions, func => _.has(variable, func));
+    return !!intrinsicFunctions.find(func => _.has(variable, func));
```
Comment (medikoo) [inline lib/plugins/aws/package/compile/events/sns/index.js:38]: Same here, let's use `some`

File: lib/plugins/aws/deploy/lib/createStack.js (hunk 1 above)
Comment (chrisVillanueva) [reply lib/plugins/aws/deploy/lib/createStack.js]: Nice catch.  I revised my PR based on your feedback.

File: lib/plugins/aws/package/compile/events/sns/index.js (hunk 2 above)
Comment (chrisVillanueva) [reply lib/plugins/aws/package/compile/events/sns/index.js]: Updated based on your feedback.

**Pull request merged**